resolver/resolver.cc

```diff
@@ -867,225 +858,206 @@ class ResolveConstantsWalk {
     }
 
     static vector<ast::ParsedFile> resolveConstants(core::GlobalState &gs, vector<ast::ParsedFile> trees,
```

Comment:
Note: I had to declare before defining since this code references `ResolveConstantsTaskState` which contains a `ResolveConstantsWalk` so it had to be declared _after_ `ResolveConstantsWalk`.